dyn_em/solve_em.F

```diff
@@ -3624,10 +3624,9 @@ SUBROUTINE solve_em ( grid , config_flags  &
 #endif
 
 
-     CALL microphysics_driver(                                            &
-      &         DT=dtm             ,DX=grid%dx              ,DY=grid%dy   &
+     CALL microphysics_driver(DT=dtm             ,DX=grid%dx              ,DY=grid%dy   &
```

Comment:
@arjanna 
You cleaned up some indenting, so fix the unnecessary mods to the dx and dy line